### DISCUSSION TITLE
Deprecated advanced mode is migrated to the workflow.

### DISCUSSION THREAD

**Discussion by cmj1014** [langgenius/dify#6583] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I am a software developer and I have accumulated a certain amount of knowledge about prompt words. I think the expert mode of prompt words is good. Why did I go offline? Can you discuss with me the willingness and ideas of going offline? I do think this feature is great. It can make prompt words more free and design more diverse dialogues

**Comment by crazywoola** [comment]
In a Chatflow or a Workflow type application, you can still use the advanced mode. The original one is migrated to these types.
